lib/github/github/file.ex

```diff
@@ -0,0 +1,41 @@
+defmodule BorsNG.GitHub.File do
+  @moduledoc """
+  The structure of GitHub pull request file
+  """
+
+  @type tjson :: map
```

Comment:
:thinking: well, the `tsomething` type names exist on github.ex file but aren't very common TBH. I think that just saying map on `from_json/1` is good enough

Reply:
⚡️